### PR TITLE
Homework If

### PR DESCRIPTION
**Pull request by vasluianvadim**

### CODE REVIEW

File: src/main/java/IfMethods/If_Methods.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
+package IfMethods;
+import java.util.Scanner;
+
+public class If_Methods {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
```
Comment (vasilevasluian) [inline src/main/java/IfMethods/If_Methods.java]: delete pls this row

File: src/main/java/IfMethods/If_Methods.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    static void pozitivNegativ() {
+        Scanner scanner = new Scanner(System.in);
+
+        System.out.println("Introdu numarul ");
+        int number = scanner.nextInt();
+      //  int number=0;
+
+        if (number >= 0) {
+            System.out.println("Numarul este pozitiv ");
+        } else if (number <= 0) {
```
Comment (vasilevasluian) [inline src/main/java/IfMethods/If_Methods.java:27]: delete if statement
you need only if(){ }else

File: src/main/java/IfMethods/If_Methods.java (hunk 1 above)
Comment (vasluianvadim) [reply src/main/java/IfMethods/If_Methods.java]: done

File: src/main/java/IfMethods/If_Methods.java (hunk 2 above)
Comment (vasluianvadim) [reply src/main/java/IfMethods/If_Methods.java:27]: we can keep this statement

**Pull request merged**